app/templates/sensor-switch.html

```diff
@@ -0,0 +1,9 @@
+<span class="sensor-text-value">
+	<div ng-show="sensor.data.properties.value !== undefined">
+        <span ng-show="sensor.data.properties.value === true" class="balanced">Turn On</span>
+        <span ng-show="sensor.data.properties.value === false" class="assertive">Turn Off</span>
```

Comment:
As this is the current status of the button, do not use an action verb ("Turn") to express it. Either use "Turned on/off" or simply "On/Off".